client/RelatedProducts.jsx

```diff
@@ -37,7 +37,7 @@ export default class RelatedProducts extends React.Component {
         <span>You may also like</span>
         <ProductContainer>
           {
-            products.map(product => <Product product={product} />)
+            products.map(product => <Product key={product.id} product={product} />)

```

Comment:
Should the id's be more unique? You noticed it on my PR so I figured I'd mention it as well :)

Reply:
Yes! Nice catch